rasa/src/Rasa/Ext.hs

```diff
@@ -107,10 +107,12 @@ module Rasa.Ext
   , HasBuffer
   , BufRef
   , HasEditor
-  , getText
   -- | A getter-lens over the buffer's Text as a 'Yi.Rope.YiString'. Use within a 'BufAction':
   --
   -- > txt <- use getText
+  , getText
+  -- | A getter-lens over text which is encompassed by a 'Range'
```

Comment:
This docstring (and the one orphaned above for `getText` can be removed now that they're added to the function definitions; Right now it'll be duplicated in the docs and will look funny 😋 .